homeassistant/components/hive/climate.py

```diff
@@ -81,26 +73,38 @@ def supported_features(self):
 
     def handle_update(self, updatesource):
         """Handle the new update request."""
-        if '{}.{}'.format(self.device_type, self.node_id) not in updatesource:
+        if 'Heating.{}'.format(self.node_id) not in updatesource:
```

Comment:
We need to keep the original format so that it aligns to the other Hive components for when new data is updated from the cloud to the cache from one Hive component, all the other Hive components refresh their data from the latest cached data

Reply:
like I say, there is no change

Reply:
@pvizeli  unless I am reading it wrong the change is going from:

```if '{}.{}'.format(self.device_type, self.node_id) not in updatesource: ```

to

```if 'Heating.{}'.format(self.node_id) not in updatesource:```


Likewise on the code above the following changes:

```self.data_updatesource = '{}.{}'.format(self.device_type, self.node_id) ```

changes to 

```self.data_updatesource = 'Heating.{}'.format(self.node_id) ```

Which again will not align with the other Hive components code.


So instead of the `self.device_type` in the format it is instead a static word "Heating", this will then not align with the other components for Hive